src/redux/books/books.js

```diff
@@ -0,0 +1,46 @@
+const ADD_BOOK = 'bookstore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
+
+const firstState = [
+  {
+    id: 1,
+    title: '',
+    author: '',
+    category: '',
+  },
+  {
+    id: 2,
+    title: '',
+    author: '',
+    category: '',
+  },
+  {
+    id: 3,
+    title: '',
+    author: '',
+    category: '',
+  },
+];
```

Comment:
- [ OPTIONAL ] I think here you miss understand the array of objects to make it empty but you should make it like this 
because like this you made empty objects inside the array and this way you specified the number of objects as three but we want it in dynamic way 😊🚀🔥✨

> const firstState = [ ]